Added content about guiding users towards using lightning (#827)
BitcoinDesign/Guide: guide/daily-spending-wallet/sending.md

Effectively supporting users when problems occur can build trust and confidence, and essential aspect for financial applications.

### Encouraging lightning network

> For the majority of payments a user makes, Lightning is likely to be the best option. It will be faster, more private, and cost less. An ideal scenario would be one where the user does not spend time considering whether to pay on-chain or Lightning — it's all bitcoin to them.

Reviewer: ```suggestion
Lightning is likely to be the best option for the majority of payments a user makes. It will be faster, more private, and cost less. An ideal scenario would be where the user does not spend time considering whether to pay on-chain or Lightning — it's all bitcoin to them.
```

However, this can be challenging with the variety of different payment formats between on-chain and Lightning. What happens when the user is trying to pay somebody, but the receiving party has given them an on-chain address instead of a Lightning invoice?

> If your wallet is Lightning-only, the user will not be able to proceed with making payment. However, even if your wallet allows the user to send on-chain payments, this payment could still result in a higher transaction fee than they would have incurred over Lightning. If it's in the users best interest to pay over Lightning, then let them know and help them determine what to do next.

Reviewer: ```suggestion
If your wallet is Lightning-only, the user will be unable to proceed with making payment. However, even if your wallet allows the user to send on-chain payments, this payment could still result in a higher transaction fee than they would have incurred over Lightning. If it's in the user's best interest to pay over Lightning, then let them know and help them determine what to do next.
```

{% include image-gallery.html pages = page.imagesLightning %}

> In this example, the user is scanning a QR to make a payment. This wallet recognizes it as an on-chain address. It is capable of making the on-chain payment with submarine swaps. However, that would involve a longer confirmation time and higher fee for such a small payment. It immediately pulls up a modal notification to warn the user that they will have to wait longer for the payment to settle, and pay a higher fee. It informs them they can pay instantly if they can get a different type of QR code from the sender.

Reviewer: ```suggestion
In this example, the user scans a QR to make a payment. This wallet recognizes it as an on-chain address. It is capable of making the on-chain payment with submarine swaps. However, that would involve a longer confirmation time and a higher fee for such a small payment. It immediately pulls up a modal notification to warn the user that they will have to wait longer for the payment to settle and pay a higher fee. It informs them they can pay instantly if they can get a different type of QR code from the sender.
```

> If the user has already had this problem before, they will know to ask the recipient for a different type of QR code. If not, they can use the "help me" screen to get more information.

Reviewer: ```suggestion
If the user has already had this problem, they will know to ask the recipient for a different type of QR code. If not, they can use the "help me" screen to get more information.
```
